src/applications/gi/sass/gi.scss

```diff
@@ -123,6 +123,18 @@
 }
 
 .vet-tec-programs-table {
+  .name-col {
+    width: 70%;
+  }
+  .hours-col {
+    width: 15%;
+  }
+  .tuition-col {
+    width: 15%;
+  }
+  .name-th {
+    padding-left: 2.5em;
```

Comment:
Recommend using the design system utilities to avoid adding duplicate css: 

https://design.va.gov/utilities/padding#padding-left